src/noise/noise_model.hpp

```diff
@@ -48,6 +48,9 @@ class NoiseModel {
   NoiseModel() = default;
   NoiseModel(const json_t &js) {load_from_json(js);}
 
+  // Load config from a JSON
+  void set_config(const json_t &config);
```

Comment:
I don't think this is a good way to deal with this in the noise model class. If you really want to disable simd, you should add a specific method to do so, not use a config. But i think this is not necessary if we remove the "disable_simd" option.